### ISSUE TITLE
Wyze Thermostat Support

### ISSUE THREAD

**Issue by r3dlobst3r** [ndejong5/homebridge-wyze-connected-home-op#2]
Plugin working great with all of my plugs, however the Wyze thermostat is not showing up. Hopefully an easy fix.

**Comment by ndejong5** [comment]
This plug-in doesn’t currently support the thermostat. If someone would like to add the code I would be more than willing to add it to the plug-in.

**Comment by r3dlobst3r** [comment]
I am working to port [this](https://github.com/JoshuaMulliken/ha-wyzeapi/blob/master/custom_components/wyzeapi/climate.py) over.

**Comment by ndejong5** [comment]
Ok, I also don’t have a lock to test just an fyi.

**Comment by r3dlobst3r** [comment]
Yeah, I don't either, just wanting my 2 thermostats to work with homebridge. If I can get it to work, I will submit PR.

**Comment by Z1mDMan** [comment]
This project might also help? https://github.com/shauntarves/wyze-sdk
